x-pack/test/alerting_api_integration/common/lib/get_event_log.ts

```diff
@@ -39,7 +39,7 @@ export async function getEventLog(params: GetEventLogParams): Promise<IValidated
   const supertest = getService('supertest');
 
   const spacePrefix = getUrlPrefix(spaceId);
-  const url = `${spacePrefix}/internal/event_log/${type}/${id}/_find?per_page=5000${
+  const url = `${spacePrefix}/api/event_log/${type}/${id}/_find?per_page=5000${
```

Comment:
Can we change this from `api` to something else?  I don't want someone to see this show up somewhere, and think this actually IS an API.  I think we can use arbitrary strings here, so could we use `_test` or something?

Reply:
Resolved in this commit [bd2bb7a](https://github.com/elastic/kibana/pull/155913/commits/bd2bb7af94264f1f1e8677ff04f234e1222177c6)